lib/api/root.rb

```diff
@@ -89,6 +89,10 @@ def authenticate
         end
       end
 
+      def set_localization
+        SetLocalizationService.new(User.current, env['HTTP_ACCEPT_LANGUAGE']).perform
```

Comment:
Note that we usually use the `current_user` helper inside the API, which is defined a few lines above and resolves to `User.current`